DuckDuckGo/MainViewController.swift

```diff
@@ -384,12 +384,13 @@ class MainViewController: UIViewController {
 
     @available(iOS 13.4, *)
     func handlePressEvent(event: UIPressesEvent?) {
-        currentTab?.tapLinkDestination = .currentTab
+        tabManager.tabControllerCache.forEach { $0.tapLinkDestination = .currentTab }
+
         if event?.modifierFlags.contains(.command) ?? false {
             if event?.modifierFlags.contains(.shift) ?? false {
-                currentTab?.tapLinkDestination = .newTab
+                tabManager.tabControllerCache.forEach { $0.tapLinkDestination = .newTab }
             } else {
-                currentTab?.tapLinkDestination = .backgroundTab
+                tabManager.tabControllerCache.forEach { $0.tapLinkDestination = .backgroundTab }
```

Comment:
As I understand, problem exists for every tab, on which modifier key has ben used. This forcefully resets `tapLinkDestination` on all tabs (so when key is released on a new tab, the "previous current tab" ;) state is fixed).

This seems ok, but since you had to change visibility of `tabControllerCache` and thus kind-of break encapsulation, I wonder if we should pursue a different solution, e.g. returning `tapLinkDestination ` to default in `webView(_ webView:decidePolicyFor:decisionHandler:)` after navigation has been initiated.

Reply:
One additional thought - we could even move it a bit further and remove `tapLinkDestination` entirely - keep modifier status in some way in MainVC, as key modifiers to me seem as more app-wide than tab-specific.

Reply:
I like the sound of removing `tapLinkDestination` entirely :+1: You're totally right, because this is something that now has to be changed on every active tab, it doesn't make sense to maintain state individually. I'll get a commit up for this today, thanks!